support/index.html

```diff
@@ -84,11 +84,21 @@ <h4>OME Training</h4>
             <div class="small-12 medium-4 columns">
                 <div class="card card-support">
                     <div class="card-section">
+                        <i class="border-red icon-red fa fa-link fa-2x"></i>
+                        <h4>Catch up with OME</h4>
+                        <p class="card-caption">Follow us on social media, read minutes of or listen in to an OME Team meeting.</p>
```

Comment:
Still a lingering "Team meeting" here

Reply:
Ah, it likely slipped in on the rebase. Thank you.